cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -118,6 +118,14 @@ def _create_entrance_exam(request, course_key, entrance_exam_minimum_score_pct=N
         is_entrance_exam=True
     )
 
+    # Create the entrance exam section item.
+    create_xblock(
+        parent_locator=unicode(created_block.location),
+        user=request.user,
+        category='sequential',
+        display_name='Exam Sequential - Subsection'
```

Comment:
This sequential is visually hidden, and only visible in the XML version of the course moving forward correct? My suggestion would be "Entrance Exam - Subsection" (the Section level is still author editable to whatever they prefer .)